icefoganalytics/asset-apps
# Fixup/harmonize linter and prettier rules

Normalize the "web" package eslint config to include prettier settings.
Add a basic prettier/eslint setup to the "api" package.

**NOTE: Should be merged after https://github.com/icefoganalytics/asset-apps/pull/18 to avoid conflicts to that more important PR**

Normalize code in "web" and "api" via.
In web:
`npm run lint`
in app:
`npm run lint`

## Testing instructions
Boot the app
- `dev up`
- `cd api; npm run start`
- `cd web; npm run start`

Check that the app boots and that pages through no new errors.
